opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CorePncMemberProfileActivity.java

```diff
@@ -36,9 +36,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
         if (i == android.R.id.home) {
             onBackPressed();
             return true;
-        } else if (i == R.id.action_pnc_member_registration) {
+        } else if (item.getItemId() == R.id.action_pnc_member_registration) {
```

Comment:
`item.getItemId()` is already assigned to `i` use the `i` directly. I would prefer if `i` was renamed to something else like `item`